### PR TITLE
Fixed issue #850 & some gradle stuff

### PR DESCRIPTION
**Pull request by crzyBrown**
Related issues: 850

Description:
Fixed issue #850, upgraded gradle wrapper from 3.4.1 to 4.6 and converted some deprecated gradle codes to new gradle codes.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,37 +34,37 @@ repositories {
...
 
 dependencies {
-	compile 'com.destroystokyo.paper:paper-api:1.12.1-R0.1-SNAPSHOT'
-	compile 'org.eclipse.jdt:org.eclipse.jdt.annotation:1.1.0'
-	compile 'com.google.code.findbugs:findbugs:2.0.3'
-	compile 'com.sk89q:worldguard:6.1.1-SNAPSHOT'
-	compile 'net.milkbowl.vault:Vault:1.6.6'
-	compile 'com.github.marcelo-mason:PreciousStones:24e3d4bf67b7240ae36b32be10e99d4091938c5c'
+	implementation 'com.destroystokyo.paper:paper-api:1.12.1-R0.1-SNAPSHOT'
+    implementation 'org.eclipse.jdt:org.eclipse.jdt.annotation:1.1.0'
```
Comment (Snow-Pyon) [inline build.gradle]: Please keep the indentation concise with the Skript standards (tabs instead of spaces).

File: build.gradle [hunk 2]
```diff
@@ -132,10 +132,19 @@ publishing {
...
-	
+
 	repositories {
 		maven {
 			url "file:///itemapi_repo/"
 		}
 	}
 }
+
+/**
```
Comment (Snow-Pyon) [inline build.gradle]: Why did you use javadoc comments here? Just use normal multiline comments

Comment (crzyBrown) [reply build.gradle]: Gradle uses groovy, its okay i think?

Comment (crzyBrown) [reply build.gradle]: Okay i'll change.

File: build.gradle (hunk 1 above)
Comment (crzyBrown) [reply build.gradle]: What do you mean use tabs or spaces? (the IDE was said; do like this to me)

Comment (crzyBrown) [reply build.gradle]: I think i did, please look again.

**Comment by Snow-Pyon** [comment]
Thanks for the contribution!

**Pull request merged**